DuckDB.NET.Bindings/DuckDBNativeObjects.cs

```diff
@@ -209,7 +209,7 @@ public DuckDBInterval(int months, int days, ulong micros)
     public static explicit operator TimeSpan(DuckDBInterval interval)
     {
         (var timeSpan, var exception) = ToTimeSpan(interval);
-        return timeSpan ?? throw exception;
+        return timeSpan ?? throw exception ?? new Exception();
```

Comment:
Why is `?? new Exception();` needed? `exception` will never be null if `timeSpan` is null so can't you just add `!` ?

Reply:
The next method can be improved in .NET 6.0 by adding a conditional post-condition

```csharp
public bool TryConvert([NotNullWhen(true)] out TimeSpan? timeSpan)
```